src/test/java/api/loans/CheckOutByBarcodeTests.java

```diff
@@ -1,5 +1,6 @@
 package api.loans;
 
+import static api.requests.RequestsAPICreationTests.setupDeclaredLostItem;
```

Comment:
I think we should consolidate these shared methods by extracting them from the individual test classes.

Reply:
@marcjohnson-kint Could you please provide more comments about your idea?

Reply:
@bohdan-suprun Done.